### ISSUE TITLE
Not completely rendering the colorizer

### ISSUE THREAD

**Issue by 3ndl3ss-l00p** [RRUZ/Delphi-IDE-Colorizer#12]
Hi,

I am using Win 10 and D10 Seattle and the colorizer does not accommodate the Structure and Object Inspector completely. Thanks

![structure](https://cloud.githubusercontent.com/assets/16831365/12503157/750f440a-c10c-11e5-8a0b-32e712e51ece.png)

**Comment by RRUZ** [comment]
Please attach the *settings.ini* file located on the folder _%LOCALAPPDATA%\The Road To Delphi\DIC\DXSeattle_

**Comment by 3ndl3ss-l00p** [comment]
Here you go. Thanks.

[Settings - Copy.zip](https://github.com/RRUZ/Delphi-IDE-Colorizer/files/101622/Settings.-.Copy.zip)

**Comment by RRUZ** [comment]
The file looks fine, Have you modified the *HookedWindows.dat* file?

**Comment by 3ndl3ss-l00p** [comment]
No, what I did was just download the installer, install it with default settings, run D10 and that is what I got. If you need, here is my HookedWindows.dat file.
[HookedWindows.zip](https://github.com/RRUZ/Delphi-IDE-Colorizer/files/104446/HookedWindows.zip)

**Comment by RRUZ** [comment]
Can you please confirm if that happens with the new version 0.8.7.1?

**Comment by 3ndl3ss-l00p** [comment]
Hi,

Yes it still does. Odd thing though is that it only happens on my custom desktop speedsetting. Other layouts are perfect. The odd one is better now than what it looked like previously.

![dic_0871](https://cloud.githubusercontent.com/assets/16831365/15648064/80b25ed4-269a-11e6-97c6-f9774c9aef82.png)
